specification/mysql/resource-manager/readme.md

```diff
@@ -257,6 +257,65 @@ suppressions:
+- Microsoft.DBforMySQL/Backups/preview/2023-06-01-preview/Backups.json
+- Microsoft.DBforMySQL/Backups/preview/2023-06-01-preview/BackupAndExport.json
+- Microsoft.DBforMySQL/Configurations/preview/2023-06-01-preview/Configurations.json
+- Microsoft.DBforMySQL/Databases/preview/2023-06-01-preview/Databases.json
+- Microsoft.DBforMySQL/Firewall/preview/2023-06-01-preview/FirewallRules.json
+- Microsoft.DBforMySQL/FlexibleServers/preview/2023-10-01-preview/FlexibleServers.json
+- Microsoft.DBforMySQL/LogFiles/preview/2023-06-01-preview/LogFiles.json
+- Microsoft.DBforMySQL/ServiceOperations/preview/2023-06-01-preview/ServiceOperations.json
+- Microsoft.DBforMySQL/FlexibleServers/preview/2023-10-01-preview/AdvancedThreatProtectionSettings.json
+- Microsoft.DBforMySQL/common-types/v1/common-types.json
+suppressions:
+  - code: PostOperationAsyncResponseValidation
+    from: FlexibleServers.json
+    reason: This check is optional.
+```
+
+### Tag: package-flexibleserver-2023-12-01-preview
+
+These settings apply only when `--tag=package-flexibleserver-2023-12-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-flexibleserver-2023-12-01-preview'
+input-file:
+- Microsoft.DBforMySQL/AAD/preview/2023-06-01-preview/AzureADAdministrator.json
+- Microsoft.DBforMySQL/Backups/preview/2023-10-01-preview/Backups.json
+- Microsoft.DBforMySQL/Backups/preview/2023-10-01-preview/BackupAndExport.json
+- Microsoft.DBforMySQL/Backups/preview/2023-10-01-preview/LongRunningBackups.json
+- Microsoft.DBforMySQL/Configurations/preview/2023-06-01-preview/Configurations.json
+- Microsoft.DBforMySQL/Databases/preview/2023-06-01-preview/Databases.json
+- Microsoft.DBforMySQL/Firewall/preview/2023-06-01-preview/FirewallRules.json
+- Microsoft.DBforMySQL/FlexibleServers/preview/2023-12-01-preview/FlexibleServers.json
+- Microsoft.DBforMySQL/FlexibleServers/preview/2023-12-01-preview/AdvancedThreatProtectionSettings.json
+- Microsoft.DBforMySQL/LogFiles/preview/2023-06-01-preview/LogFiles.json
+- Microsoft.DBforMySQL/ServiceOperations/preview/2023-12-01-preview/ServiceOperations.json
+- Microsoft.DBforMySQL/Maintenance/preview/2023-10-01-preview/Maintenances.json
+suppressions:
+  - code: PostOperationAsyncResponseValidation
+    from: FlexibleServers.json
+    reason: This check is optional.
+  - code: PutResponseCodes
+    from: LongRunningBackups.json
+    reason: "202 is a pattern that is already used in our existing resources and being carried forward to new implementations to maintain consistency for our customers. This has already been approved by the API review board."
+  - code: PutResponseCodes
+    from: AdvancedThreatProtectionSettings.json
+    reason: "202 is a pattern that is already used in our existing resources and being carried forward to new implementations to maintain consistency for our customers. This has already been approved by the API review board."
+  - code: PutInOperationName
```

Comment:
Why do you need to keep Put word in the operation name for newly added API? You can update the operation id to use "_CreateOrUpdate" instead of using "_Put" or "_UpdatePut" etc. in new APIs.

Reply:
This is not operation name, but the linter error code. we need to suppress this linter error so we put it in readme suppression